core/src/main/java/org/apache/calcite/rex/RexAnalyzer.java

```diff
@@ -41,6 +43,16 @@
   public final List<RexNode> variables;
   public final int unsupportedCount;
 
+  public static final EnumSet<SqlKind> SUPPORTED_SQL_KIND =
+      EnumSet.of(SqlKind.IS_NOT_DISTINCT_FROM, SqlKind.EQUALS, SqlKind.IS_DISTINCT_FROM,
```

Comment:
Does it belong to `RexInterpreter`?

Reply:
I am not very strong about it. Will Remove it to `RexInterpreter`.